### ISSUE TITLE
Incorrect Brotli example

### ISSUE THREAD

**Issue by matthewdouglas** [webpack-contrib/compression-webpack-plugin#154]
Documentation Is:

- [ ] Missing
- [x] Needed
- [x] Confusing
- [ ] Not Sure?

### Please Explain in Detail...
The Brotli examples in the README do not correctly set the `compressionOptions`. The `BROTLI_PARAM_QUALITY` parameter is the equivalent to zlib's `level` option in terms of functionality, but not naming.

### Your Proposal for Changes
1. Revise the examples to set the Brotli parameters correctly.

```js
const zlib = require('zlib');
...
compressionOptions: {
  params: {
    [zlib.constants.BROTLI_PARAM_QUALITY]: 11
  }
}
```

2. Update the note to be more clear about `BROTLI_PARAM_QUALITY` being functionally equivalent to `level`, but not syntactically. 

3. Update note regarding Node version as Brotli native support was backported to [Node 10.16](https://github.com/nodejs/node/pull/27681).

**Comment by evilebottnawi** [comment]
Feel free to send a PR with fixes

**Comment by 180254** [comment]
Points 1 and 2 described there are still valid and documentation needs to be improved.

Rationale:
- Look at the typescript declaration for node zlib module (https://unpkg.com/browse/@types/node@14.14.2/zlib.d.ts).  ZlibOptions interface has the "level" property, while BrotliOptions does not.
- Look at the node zlib module source code (https://github.com/nodejs/node/blob/v15.0.1/lib/zlib.js). Brotli (line 793) do not have the "level" property.
- This plug-in (compression-webpack-plugin) has no additional conditions in the code to support this syntax.
- I created reproduction-steps (https://github.com/180254/compression-webpack-plugin-issue-154) that shows that master-branch-example does not work. I compressed a file several times. Changing "level" has no effect, while changing BROTLI_PARAM_QUALITY has.
```java
$ ls -lA dist/
-... 89930 paź 24 20:38 jquery.js
-... 28094 paź 24 20:38 jquery.js-compressionOptions-level-0.br
-...28094 paź 24 20:38 jquery.js-compressionOptions-level-11.br
-... 37188 paź 24 20:38 jquery.js-compressionOptions-params-BROTLI_PARAM_QUALITY-0.br
-... 28094 paź 24 20:38 jquery.js-compressionOptions-params-BROTLI_PARAM_QUALITY-11.br

```
